zipline/__init__.py

```diff
@@ -12,6 +12,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
+
+
+if os.name == 'nt':
```

Comment:
I think so; if this is making all the tmp files go to this directory, it is necessary. The tmp files have auto-delete turned off, so if they're ending up here, it's necessary for cleanup